tests/integration/models/api-key.spec.ts

```diff
@@ -124,6 +124,33 @@ describe('API Key model', function () {
 		});
 	});
 
+	describe('balena.models.apiKey.getProvisioningApiKeysByApplication', function () {
+		givenLoggedInUser(before);
+
+		before(function () {
+			this.appName = 'SampleOrg/sampleApp';
+			return balena.models.application
+				.generateApiKey('SampleOrg/sampleApp')
+				.then(function (apiKey) {
```

Comment:
```suggestion
				.then((apiKey) => {
```
or convert to async-await, b/c the `this` in here is probably not what we think.